sky/core.py

```diff
@@ -529,6 +529,7 @@ def cancel(
     cluster_name: str,
     all: bool = False,
     job_ids: Optional[List[int]] = None,
+    cancel_latest_running_job: bool = False,
```

Comment:
I am thinking if we should change the argument to be `cancel_all: bool = True` to be consistent with the CLI, i.e., `job_ids = None` and `cancel_all=False` means to cancel the latest running job.